### DISCUSSION TITLE
Segmentation fault on myMPd 13.0.0

### DISCUSSION THREAD

**Discussion by WallyDW** [jcorporation/myMPD#1145] [Q&A]
I have installed version 13.0.0 of myMPD and connected it to a MPD version 0.23.12-1+b1 service on a Debian 12 system.
Everytime I click on a picture in the database it will die with a segmentation fault and it will print a message in top that about being disconnected from MPD and trying to reconnect and in the track listing it will display "Error posting to /api/default".

I have looked in the logs, but found no error there and I have had a debug socket up and no clue there either.
I had sadly no possibility to run a debug build, so I was hoping someone else have experienced the same and could give a hint how to solve it.

It has to be mentioned that everything else works as long as I do not have to click the database picture.
That means I can add songs to playlists with the text views.
I can play songs and get the background picture and the album picture in the Playback view.

**Comment by jcorporation** [comment]
Increase the Loglevel to Debug. Why do you not have the possibility to run a debug build? You can install the debug symbols.

Any none-custom configuration and the platform is also interesting.

**Comment by WallyDW** [comment]
I already ran it with a loglevel set to debug, but the segmentation fault is all that it writes to log.
I looked into doing the debug build, but the hardware is scaled down to just fit the system and storage became a problem.
I had earlier been running myMPD version 12.1.1, which worked fine on the setup.
I tried to roll back to that version, but it was not available anymore.

**Reply by jcorporation** [reply]
With loglevel set to debug, there should be many lines what is ongoing in the log. Sure your logging is configured correctly? 

Without logs, detailed config and system infos and stacktrace nobody can help you.

**Reply by jcorporation** [reply]
It seems you had luck and an other mpd user reported the problem with usefull logs: https://github.com/jcorporation/myMPD/issues/1151
